test/e2e/kubernetes/deployment/deployment.go

```diff
@@ -117,7 +117,9 @@ func CreateLinuxDeploy(image, name, namespace, miscOpts string) (*Deployment, er
 	var commandTimeout time.Duration
 	var cmd *exec.Cmd
 	overrides := `{ "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}}`
-	if miscOpts != "" {
+	if miscOpts == "1.16+" {
+		cmd = exec.Command("k", "create", "deployment", name, "-n", namespace, "--image", image)
```

Comment:
One problem with just switching to `kubectl create deployment` is that there is no `--overrides` flag to customize the deployment created. So don't have a way to specify a `nodeSelector` that ensures a workload lands on a Linux node. I think we may have to explicitly create a deployment manifest to get that behavior.

There are two other places in `deployment.go` that do `kubectl run` and then wait for a deployment to show up, so those would need to be fixed as well.